client-grpc/Cargo.toml

```diff
@@ -18,4 +20,4 @@ name = "svc_scheduler_client_grpc"
 path = "src/lib.rs"
 
 [[example]]
-name = "grpc"
+name = "client"
```

Comment:
I think it's better to keep this name to "grpc" since we use that for all examples that are written for the client-grpc code and it's named like that in our template repo too.